src/content/deployment/_index.en.md

```diff
@@ -52,5 +52,5 @@ For each cluster you want to join, issue the following command:
 subctl join --kubeconfig <PATH-TO-JOINING-CLUSTER> broker-info.subm --clusterid <ID>
 ```
 
-subctl will automatically discover as much as it can, and prompt the user for any missing necessary information. Note that each cluster must
-be configured with a unique cluster ID.
+`subctl` will automatically discover as much as it can, and prompt the user for any missing necessary information. Note that each cluster must
+have a unique cluster ID; the cluster ID can be specified, or otherwise is going to be generated by default based on the cluster name.
```

Comment:
```suggestion
have a unique cluster ID; the cluster ID can be specified, or otherwise is going to be generated by default based on the cluster name in the `kubeconfig` file.
```